core/hoverfly.go

```diff
@@ -241,8 +241,15 @@ func (hf *Hoverfly) captureRequest(req *http.Request) (*http.Response, error) {
 
 	if err != nil {
 		log.WithFields(log.Fields{
-			"error": err.Error(),
-			"mode":  "capture",
+			"error":       err.Error(),
+			"mode":        "capture",
+			"Path":        req.URL.Path,
+			"Method":      req.Method,
+			"Destination": req.Host,
+			"Scheme":      req.URL.Scheme,
+			"Query":       req.URL.RawQuery,
+			"Body":        string(reqBody),
```

Comment:
you will still get `panic` here. If an error occurs - probably just add something like:
`reqBody = []byte(") `
and log as much as possible (like you are doing now). Also, parameters from req.TLS struct could be useful as well for debugging